content/essentials/intro-to-reactivity.mdx

```diff
@@ -0,0 +1,169 @@
+import { Aside } from "~/components/configurable/Aside";
+
+# Intro to Reactivity
+
+**Note**: While this guide is useful for understanding reactive systems, it does use some Solid-specific terminology.
+
+Reactivity, at its core, provides the groundwork to power interactivity in dynamic applications. 
+This programming paradigm is when systems automatically respond to changes in data or state, ensuring that the application's interface stays consistent with its underlying data. With Solid, reactivity is the basis of its design. 
+
+## Importance of reactivity
+
+1. **Automatic UI updates**: Reactivity ensures that as data changes, the user interface is automatically updated to reflect those changes.
+
+2. **Simplified code**: By removing the need for manual DOM updates or UI refreshes, reactivity reduces the need for manual interventions. 
+This synchronizes the state and the UI, leading to more straightforward and maintainable code.
+
+3. **Consistent user experience**: With the UI always being in sync with the underlying data, users get a consistent and expected response to their actions.
+
+4. **Scalability**: As applications grow, managing state and UI can become complex. Reactivity abstracts this complexity, ensuring that even large-scale applications remain manageable.
+
+## Reactive principles
+
+When delving into the world of reactivity, you'll encounter a few foundational principles:
+
+### Signals
+
+Signals serve as foundational elements in reactive systems, playing a pivotal role in data management and system responsiveness.
+
+A signal can be visualized as a container for data. 
+It not only holds the data, but also provides mechanisms for accessing and modifying its value, a **getter **and a **setter**. 
+This dual capability makes signals both readable and mutable, making them integral to the dynamic nature of reactive systems.
+
+#### Getter
+
+The getter is the mechanism responsible for accessing the current value of the signal. 
+When various components or parts of an application need to "read" or retrieve the data stored in a signal, they use the getter. 
+This provides data that is consistent and reliably accessible throughout the system.
+
+#### Setter
+
+On the other hand, the setter is the tool that permits the changing of the signal's value. 
+Any change to the value within a signal, made possible by the setter, can instigate reactive updates in other connected parts of the system. 
+This ensures that the system remains responsive and up-to-date with the latest data changes.
+
+### Effects
+
+Effects can be thought of as automated responder. 
```

Comment:
Maybe something like
> Effects and other listeners (such as memos) can be thought of as automated responders.

because effects are not special in this regard.